src/components/add-products/add-products.js

```diff
@@ -24,10 +40,16 @@ function AddProducts() {
 
   const onSubmit = (event) => {
     setSubmitted(true);
-    if (title && price && categoryId && imageUrl) {
+    if (id === 0 && title && price && categoryId && imageUrl) {
```

Comment:
since all of the conditions after `id===0` are the same for both, it should probably be its own block that encompasses  the whole logic

Reply:
In fact, it appears that you would want to exit out of the function if those conditions aren't met. You may want to consider using
`if (!(title && price && categoryId && imageUrl)) return;`
This will break out of the method and minimize the nested `if` blocks